README.md

```diff
@@ -78,25 +76,30 @@ if (obj1.sketchObject == obj2.sketchObject) {
 Here's a very simple example script:
 
 ```javascript
-var sketch = context.api()
+var api = SketchAPI
 
-log(sketch.api_version)
-log(sketch.version)
-log(sketch.build)
-log(sketch.full_version)
+log(api.version.api)
+log(api.version.sketch)
 
-var document = sketch.selectedDocument
+var document = api.fromNative(context.document)
 var selection = document.selectedLayers
 var page = document.selectedPage
 
-var group = page.newGroup({
+var Group = api.Group
+var Shape = api.Shape
+
+var group = new Group({
+  parent: page,
   frame: new sketch.Rectangle(0, 0, 100, 100),
```

Comment:
for the purposes of this example, that should be `new api.Rectangle` I think?

Reply:
:+1: